src/style.css

```
@@ -0,0 +1,32 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  margin-right: 4px;
+  height: 16px;
+  width: 16px;
+  background: center center no-repeat url(./images/star.svg);
+}
+
+.stars--1 > div:nth-of-type(1),
+.stars--2 > div:nth-of-type(1),
+.stars--2 > div:nth-of-type(2),
+.stars--3 > div:nth-of-type(1),
+.stars--3 > div:nth-of-type(2),
+.stars--3 > div:nth-of-type(3),
+.stars--4 > div:nth-of-type(1),
+.stars--4 > div:nth-of-type(2),
+.stars--4 > div:nth-of-type(3),
+.stars--4 > div:nth-of-type(4),
+.stars--5 > div:nth-of-type(1),
+.stars--5 > div:nth-of-type(2),
+.stars--5 > div:nth-of-type(3),
+.stars--5 > div:nth-of-type(4),
+.stars--5 > div:nth-of-type(5) {
+  background: center center / 13px 12px no-repeat url(./images/star-active.svg);
+}

```

```suggestion
.stars--5 .stars__star,
.stars--1 .stars__star:first-child,
.stars--2 .stars__star:not(:nth-of-type(n+3)),
.stars--3 .stars__star:not(:nth-of-type(n+4)),
.stars--4 .stars__star:not(:last-child) {
  background: url("images/star-active.svg") no-repeat center;
}
```
Давай сильно спростимо цей код

---

Ок.
Тобто `.stars--2 .stars__star:not(:nth-of-type(n+3)) = .stars--2 .stars__star:nth-of-type(-n+2)`, зрозумів, дякую.